registries/princeton/documents.csv

```diff
@@ -7,3 +7,26 @@
 "PFS-ICS-PRU030000-01", "PFS ICS Hardware Configuration Report", "Hassan Siddiqui", "https://sumire.pbworks.com/w/file/126994364/PFS-ICS-PRU030000-01_PFS_ICS_Hardware_Configuration_Report.pdf", "2018-05-07", 
 "PFS-DRP-PRU050001-01", "Interim report on the status of 2D PSF modelling", "Neven Caplar", "https://sumire.pbworks.com/w/file/fetch/127999977/PFS-DRP-PRU050001-01_2D_modelling_status.pdf", "2018-06-26",
+"PFS-GEN-PRU000010-01", "Uploading Whitepaper HOWTO", "Jim Gunn", "", "2011-01-01", "howto dummies"
+"PFS-PFI-PRU000020-01", "Summary of Subaru/WFC Imaging Performance", "Jim Gunn", "", "2012-04-04", "Summary of Subaru/WFC imaging performance for PFS with vigetting, angles, etc."
+"PFS-PFI-PRU000030-01", "Notes on rotator and A&G cameras", "Jim Gunn", "", "2011-12-03", "Notes on the rotator and A&G cameras and rotation limits"
+"PFS-PFI-PRU000040-01", "Notes on geometry of fiber slit", "Jim Gunn", "", "2012-01-07", "Notes on the geometry of the fiber slit"
+"PFS-PFI-PRU000050-01", "Notes on dither", "Jim Gunn", "", "2012-05-17", "Notes on the requirements for the dither mechanism for the FSA"
+"PFS-PFI-PRU000060-01", "Notes on aiming fibers", "Jim Gunn", "", "2013-09-12", "Notes on aiming the fibers to create a fiber pupil in the right place (on the gratings) AND the detailed geometry for the PFS slit masks.  Latest version (130912) has *hopefully* correct detector geometry and new slit curvature. PLEASE discard ALL earlier versions."
+"PFS-PFI-PRU000070-01", "Notes on the formation of ghosts by glass slit plate", "Jim Gunn", "", "2012-09-10", "Notes on the formation of ghosts by a glass slit plate"
+"PFS-PFI-PRU000080-01", "Fiber In/Out", "Jim Gunn", "", "2012-08-22", "Notes on aiming the fibers to create a fiber pupil in the right place (on the gratings) AND the detailed geometry for the PFS slit masks.  Latest version (130912) has *hopefully* correct detector geometry and new slit curvature. PLEASE discard ALL earlier versions."
+"PFS-FIB-PRU000090-01", "Fiber FRD", "Jim Gunn", "", "2013-04-11", "Modelling the fiber FRD using the LNA experiments"
+"PFS-FIB-PRU000100-01", "Fiber Routing", "Jim Gunn", "", "2012-10-02", "Suggestion for fiber routing with no crossovers."
+"PFS-PFI-PRU000110-01", "PFS Focus", "Jim Gunn", "", "2012-11-18", "A study on the location of the PFS focal plane with respect to the WFC, and a short study on guiding, focussing, and aquisition errors."
+"PFS-FIB-PRU000120-01", "Crosstalk", "Jim Gunn", "", "2013-03-21", "A proposed definition of fiber-to-fiber crosstalk and a proposed specification"
+"PFS-PFI-PRU000130-01", "WFC Errors", "Jim Gunn", "", "2013-04-09", "A small study on the effect of the observed high-frequency slope errors in the WFC on Metrology Camera performance"
+"PFS-PFI-PRU000140-01", "Back Illumination", "Jim Gunn", "", "2013-04-16", "A small study on what illumination level is needed for the fiber back-illumination."
```

Comment:
Category : GEN?
Back illumination relates with both PFI (fiducial fibers) and SpS (science fibers), so I think "general" would be proper... what do you think of it?